cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
-    {% load i18n %}
+    {% load i18n third_party_auth_links %}
     {% load compressed %}
     {% compressed_css 'style-main' %}
+    {% compressed_css 'style-vendor' %}
```

Comment:
style-vendor always needs to come before style-main. please move this above style-main.